docs/getting_started/configuration/prawini.rst

```diff
@@ -145,7 +145,7 @@ This uses basic interpolation thus :class:`.Reddit` need to be initialized as fo
 
 .. code-block:: python
 
-    reddit = praw.Reddit("bot1", config_interpolation="basic")
+    reddit = praw.Reddit(site_name="bot1", config_interpolation="basic")
```

Comment:
These aren't sorted (if that was the intent behind the commit).

Reply:
That is because it's a defined argument. For some reason, PyCharm decided to sort them by defined arguments then arbitrary arguments.